README.md

```diff
@@ -86,6 +86,28 @@ static_metadata:
   * All `static_metadata` entries must have `type` specified. This specifies the Stackdriver metric type and overrides the metric type chosen by the Prometheus client.
   * If `value_type` is specified, it will override the default value type for counters and gauges. All Prometheus metrics have a default type of double.
 
+#### Dealing with recording rules
+
+The default Prometheus naming format for [recording rules](https://prometheus.io/docs/practices/rules/) is `level:metric:operations`, e.g. `instance:requests_total:sum`, but colons are not allowed in Stackdriver metric descriptor names.  To forward a recorded Prometheus metric to Stackdriver, you must use the `metric_renames` feature to replace the colon characters:
```

Comment:
Still missing the comma after `e.g.`. Also in line 99 below.

Granted, it's missing one in line 64 as well, so maybe I'm being too pedantic…

Reply:
I'm 100% here for punctuation pedantry -- I'll go and make it consistent throughout.